Ports/FFmpeg/package.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=ffmpeg
+version=4.3.2
+useconfigure=true
+files="https://ffmpeg.org/releases/ffmpeg-${version}.tar.xz ffmpeg-${version}.tar.xz
+https://ffmpeg.org/releases/ffmpeg-${version}.tar.xz.asc ffmpeg-${version}.tar.xz.asc"
+makeopts="HOSTCC=${CC} HOSTLD=${CC}"
+depends="SDL2"
+
+configure() {
+	run ./configure --enable-cross-compile --target-os=serenity --arch=i686 \
+	    --enable-ffmpeg --enable-ffprobe --enable-ffplay --cross-prefix=${CC%gcc} \
+	    --disable-txtpages --disable-manpages --disable-podpages \
+	    --disable-static --enable-shared \
+	    --enable-debug --disable-stripping # Remove Before PR!
+}
```

Comment:
You should be able to move all of these into a `configopts` parameter.

```suggestion
configopts="--enable-cross-compile --enable-ffmpeg --enable-ffprobe --enable-ffplay --cross-prefix=${CC%gcc} --disable-txtpages --disable-manpages --disable-podpages --disable-static --enable-shared --enable-debug --disable-stripping"
```

`configopts` is already defined in `.port_include.sh` and sets the `host` flag automatically:

```
--host="${SERENITY_ARCH}-pc-serenity"
```

I think you can also drop the ` --target-os=serenity --arch=i686` flags. This should be picked up by the configure script automatically (maybe).

Also, unless generating the man pages is causing issues, you can remove `--disable-manpages`. It doesn't hurt to have the man pages installed. Many other ports do this, and maybe we'll one day have a port of `man` to read man pages in groff format.